spring-auto-restdocs-core/src/test/java/capital/scalable/restdocs/javadoc/JavadocUtilTest.java

```diff
@@ -108,17 +108,49 @@ public void convertBulletedListMarkdown() {
     }
 
     @Test
-    public void convertStylingAsciidoc() {
-        String actual = "<b>bold</b>normal<i>italics</i>";
-        String expected = "**bold**normal__italics__";
+    public void convertBoldStylingAsciidoc() {
+        String actual = "before<b>bold</b>normal<strong>bold2</strong>after";
+        String expected = "before**bold**normal**bold2**after";
         assertThat(JavadocUtil.convertFromJavadoc(actual, TemplateFormatting.ASCIIDOC),
                 is(expected));
     }
 
     @Test
-    public void convertStylingMarkdown() {
-        String actual = "<b>bold</b>normal<i>italics</i>";
-        String expected = "**bold**normal*italics*";
+    public void convertItalicsStylingAsciidoc() {
+        String actual = "before<i>italics</i>normal<em>italics2</em>after";
+        String expected = "before__italics__normal__italics2__after";
```

Comment:
would this even work in asciidoc?

Reply:
Yes:
<img width="198" alt="image" src="https://user-images.githubusercontent.com/1205697/79326326-f7197100-7f12-11ea-9c3f-8967bd89b39a.png">
via https://asciidoclive.com/